core-bundle/src/Resources/config/services.yml

```diff
@@ -69,6 +69,31 @@ services:
             - '%kernel.project_dir%'
         public: true
 
+    Contao\CoreBundle\Controller\BackendPreviewController:
```

Comment:
The new services need to be tested in the `ContaoCoreExtensionTest` class.

Reply:
Done in 550a56124dd8073734a8b40d8a1ad1b232c38e31.